### PR TITLE
Update the readme to be up to date with the repo content

### PR DESCRIPTION
**Pull request by jteijema**
This PR updates the repo readme with new information.

### CODE REVIEW

**Comment by jteijema** [comment]
See https://github.com/JTeijema/semantic-clusters/tree/readme-update for a easier review.

**Review by Rensvandeschoot — APPROVED** [review]

**Comment by Rensvandeschoot** [comment]
Nice work! I have added some hyperlinks and contact information.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,53 @@
-# Semantic Clusters
-Experimental repository aimed at using transformers (such as CovidBERT) and Deep Learning techniques to retrieve and visualize semantic clusters underlying the CORD-19 database.
+# ASReview Semantic Clustering
+This repository contains the Semantic Clustering plugin for [ASReview](https://github.com/asreview/asreview). It applies
+multiple techniques (SciBert, PCA, T-SNE, KMeans, a custom Cluster Optimizer) to
+an [ASReview data object](https://asreview.readthedocs.io/en/latest/API/reference.html?highlight=data%20object#asreview-data), in order to cluster records based on semantic
```
Comment (J535D165) [inline README.md]:
```suggestion
an [ASReview data object](https://asreview.readthedocs.io/en/latest/API/reference.html#asreview-data), in order to cluster records based on semantic
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
 
 ![Alt Text](/docs/cord19_semantic_clusters.gif)
+
+## Usage
+The usage of the semantic clustering app is found in the main.py file. The
+following commands can be run:
+
+### Processing
+```console
+py asreviewcontrib\semantic_clustering\main.py -f <url or local file>
```
Comment (J535D165) [inline README.md]:
```suggestion
python asreviewcontrib\semantic_clustering\main.py -f <url or local file>
```

Comment (J535D165) [reply README.md]: Also for the others.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+```
+
+### Processing testfile
+```console
+py asreviewcontrib\semantic_clustering\main.py -t
+py asreviewcontrib\semantic_clustering\main.py --testfile
+```
+
+This argument will start the processing file using the [`van_de_Schoot_2017`
+dataset](https://asreview.readthedocs.io/en/latest/intro/datasets.html?highlight=ptsd#featured-datasets), and can be used as a functionality test.
```
Comment (J535D165) [inline README.md]:
```suggestion
dataset](https://asreview.readthedocs.io/en/latest/intro/datasets.html#featured-datasets), and can be used as a functionality test.
```

**Review by J535D165 — CHANGES_REQUESTED** [review]

**Review by Rensvandeschoot — APPROVED** [review]

**Review by Rensvandeschoot — APPROVED** [review]

**Pull request merged**